### DISCUSSION TITLE
Lack of attention to dotnet/wpf repo

### DISCUSSION THREAD

**Discussion by wstaelens** [dotnet/runtime#70729] [General]
When reading this, it looks like we got your attention. 😺 🎣 

👋 Hey runtime

Maybe some folks from runtime can have a look at [WPF](https://github.com/dotnet/wpf/), the community asks for attention, creates PR's, but nothing happens.  If it gets no attention developers will drop out....

For frustrations, see:
https://github.com/dotnet/wpf/discussions/6542#discussioncomment-2947498 

😢 😞

**Comment by GeraudFabien** [comment]
I posted this https://github.com/dotnet/wpf/issues/3904 has a joke in the past. Because it was abandon like [XAML-Standard](https://github.com/Microsoft/xaml-standard/issues/230). But i really believe MS should have officially killed WPF, WinForm and webForm. This way no one would be expecting anythinks.  It's not like you app will not work. It's just that there will be no more update.

So let me rephrase myself. WPF is dead. Use Uno or blazor or avalonia. Avoid MAUI has much has possible it is already dying. And javascript, java any ecosystem lost a lot of framework to progress. Why the .net ecosystem cannot accept the death of a few.

Please do not pollute other repo. The commit history of WPF seems (to me) to have more activity this year (Not counting dotnet-maestro and documentation commit) than the last.

**Reply by wstaelens** [reply]
WPF and WinForms are currently the only mature frameworks (not talking about asp.net / asp.net core).

We are currently working with Blazor and MAUI and it is not enterprise ready (bugs, crashes, ...) and my guess is that it will still take a couple of years before it is mature. Even .NET 6 is missing things from .NET Framework. .NET itself is doing fine, but if you go into a niche like Printing you will notice it is not as mature/stable as .NET Framework.

Because of this, WPF (and WinForms) still deserve more attention and more updates/fixes etc...

When you look at component builders (e.g. DevExpress) you'll notice that the status of MAUI and Blazor components are preview/beta/... or missing features compared to similar controls in a different technology, to confirm this.
